packages/consumption/src/consumption/CoreErrors.ts

```diff
@@ -15,54 +15,54 @@ class Attributes {
     public successionMustNotChangeKey() {
         return new CoreError(
             "error.consumption.attributes.successionMustNotChangeKey",
-            "The predecessor attribute's key does not match that of the successor. The succession of a relationship attribute must not change the key."
+            "The predecessor Attribute's key does not match that of the successor. The succession of a RelationshipAttribute must not change the key."
         );
     }
 
     public successionPeerIsNotOwner() {
-        return new CoreError("error.consumption.attributes.successionPeerIsNotOwner", "The peer of the succeeded attribute is not its owner. This may be an attempt of spoofing.");
+        return new CoreError("error.consumption.attributes.successionPeerIsNotOwner", "The peer of the succeeded Attribute is not its owner. This may be an attempt of spoofing.");
     }
 
     public predecessorSourceAttributeIsNotRepositoryAttribute() {
-        return new CoreError("error.consumption.attributes.predecessorSourceAttributeIsNotRepositoryAttribute", "Predecessor source attribute is not a repository attribute.");
+        return new CoreError("error.consumption.attributes.predecessorSourceAttributeIsNotRepositoryAttribute", "Predecessor sourceAttribute is not a RepositoryAttribute.");
     }
 
     public successorSourceAttributeIsNotRepositoryAttribute() {
-        return new CoreError("error.consumption.attributes.successorSourceAttributeIsNotRepositoryAttribute", "Successor source attribute is not a repository attribute.");
+        return new CoreError("error.consumption.attributes.successorSourceAttributeIsNotRepositoryAttribute", "Successor sourceAttribute is not a RepositoryAttribute.");
     }
 
     public successorSourceDoesNotSucceedPredecessorSource() {
         return new CoreError(
             "error.consumption.attributes.successorSourceDoesNotSucceedPredecessorSource",
-            "Predecessor source attribute is not succeeded by successor source attribute."
+            "Predecessor sourceAttribute is not succeeded by successor sourceAttribute."
         );
     }
 
     public predecessorSourceContentIsNotEqualToCopyContent() {
         return new CoreError(
             "error.consumption.attributes.predecessorSourceContentIsNotEqualToCopyContent",
-            "Predecessor source attribute content doesn't match predecessor shared attribute copy."
+            "Predecessor sourceAttribute content doesn't match predecessor shared Attribute copy."
```

Comment:
"Predecessor sourceAttribute's content does not match **the content of the** predecessor shared Attribute copy."